pytest_selenium/pytest_selenium.py

```diff
@@ -198,6 +198,8 @@ def pytest_runtest_makereport(item, call):
                 item=item, report=report, extra=extra)
         item.config.hook.pytest_selenium_runtest_makereport(
             item=item, report=report, summary=summary, extra=extra)
+    # always gather driver logs
+    _gather_driver_logs(item, summary, extra)
```

Comment:
We should only do this if `capture_debug` is True. Perhaps swap the `driver is not None` and `capture_debug` if blocks around.

Reply:
This is back to the original issue. We should only gather the driver logs if `capture_debug` is True. We just also need to call `item.config.hook.pytest_selenium_runtest_makereport` regardless of the value of `capture_debug`.

Reply:
Is that why you said to switch the order of them?

Reply:
Yeah, I just thought that would be cleaner. At the time I was forgetting about the hook, but I think this is fine outside of the driver check. Something like:

```python
if capture_debug:
    _gather_driver_logs(item, summary, extra)
    if driver is not None:
        # selenium related debug capturing
item.config.hook.pytest_selenium_runtest_makereport(
    item=item, report=report, summary=summary, extra=extra)

Reply:
I'll have to look over it again, I think I was getting an error with that.

Reply:
Here is a log: https://pastebin.mozilla.org/9077828